### PR TITLE
Migrate launch tests to new launch_testing features & API

### PR DESCRIPTION
**Pull request by hidmic**
Connected to ros2/launch#208.

### CODE REVIEW

**Comment by dirk-thomas** [comment]
Please use a separate commit (which isn't being squashed when merging later) to rename existing files (instead of deleting one and creating another with very similar content).

**Comment by hidmic** [comment]
> Please use a separate commit to rename existing files

Hmm, it is true that the two current tests were based upon an existing one that was later deleted. But this isn't pure renaming, the original test was split in two. Would you rather have the original test renamed to one of the current ones?

**Comment by dirk-thomas** [comment]
I just want the history to be preserved. If you want to use a different filename then you should rename the file in a separate commit without making any changes to it. Follow up commits can then change the content in any way.

**Comment by hidmic** [comment]
Done, sorry about the force push. Github's UI shows the original file as deleted though.

**Comment by dirk-thomas** [comment]
The blame view can now track changes from the past: https://github.com/ros2/ros1_bridge/blame/a96ee952c63c0975887c09057b1b6e062c881951/test/test_topics_across_dynamic_bridge.py.in

**Comment by dirk-thomas** [comment]
Looking at the diff now it seems that the service file is also derived from the original file (I though it was a new addition). In that case it would be good to copy the file after renaming it and before making the changes - again to be able to track the changes of the content.

**Comment by hidmic** [comment]
Done @dirk-thomas !

File: test/test_services_across_dynamic_bridge.py.in [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    launch_description.add_action(
+        OpaqueFunction(function=lambda context: ready_fn())
+    )
+    return launch_description, locals()
+
+
+@launch_testing.post_shutdown_test()
+class TestServicesAcrossDynamicBridgeAfterShutdown(unittest.TestCase):
+
+    def test_processes_finished_gracefully(self, proc_info, rosbridge_process,
```
Comment (dirk-thomas) [inline test/test_services_across_dynamic_bridge.py.in]: Nitpick: wrap after open parenthesis.

Same below.

**Comment by hidmic** [comment]
@jacobperron @dirk-thomas mind to approve with powers? @ivanpauno lacks them.

**Pull request merged**